web/src/containers/ExecutiveSummary.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { Section, Subsection } from '../components/Section';
+import StayTuned from '../components/StayTuned';
+import { aggregateByYear, getActivityTotals } from '../reducers/activities';
+import { formatMoney } from '../util/formats';
+
+const ExecutiveSummary = ({ data, years }) => (
+  <Section id="executive-summary" resource="executiveSummary">
+    <Subsection resource="executiveSummary.summary">
+      {data.map((d, i) => (
+        <div
+          key={d.id}
+          className="mb2 p2 sm-flex items-center alert alert-success"
+        >
+          <div className="p1 sm-m0 flex-auto">
+            {d.id !== 'all' && <div className="h5">Activity {i + 1}</div>}
```

Comment:
This `Activity {i + 1}` needs to go into the resource file, along with the `FFY {year}` below.

Reply:
Re: activity name, I think we already have a resource entry for that.  Maybe we should use that one?  Then it's consistent across locations?  Feels like a @quarterback question.